### ISSUE TITLE
Slide background radial-gradient support

### ISSUE THREAD

**Issue by nalgritz** [FormidableLabs/spectacle#557]
I tried to add gradient background color to Slide tag. As `Silde` doesn't have background props so i added as follow:
```
<Slide 
  style={{ background: 'radial-gradient(ellipse at bottom, black, white)' }}
>
  { /* other components */ }
</Slide>
```
However it only shows white background color. Also tried below return transparent background but still no radial-gradient
```
<Slide 
  bgColor='transparent'
  style={{ background: 'radial-gradient(ellipse at bottom, black, white)' }} 
>
  { /* other components */ }
</Slide>
```
I wonder if it supports gradient background?

**Comment by jalmarazg** [comment]
Hi! 
Actually I tried what you wanted and actually got a workaround. So, I'll first tell you why It's not working with the actual properties of the "Slide" tag:

### Reproducing the scenario
In this case I tried using ```bgColor``` as you suggested but got no results. After that, I tried creating a _theme_ with the ```createTheme``` method and tried with ```bgImage``` again with no results. 
I decided to check what could it be that cause the _radial-gradient_ not to show by inspecting the page and I got the following: 

- Using ```bgColor``` 
<img width="558" alt="im1" src="https://user-images.githubusercontent.com/9245962/43371063-f4aba6aa-934f-11e8-9a07-6d0b8e851fdf.png">

- Using ```bgImage``` 
<img width="286" alt="im2" src="https://user-images.githubusercontent.com/9245962/43371070-00004024-9350-11e8-893b-4cacd88ea1a9.png">

Checking the topic more in depth I came to realize that _radial-gradient_ only works if it's declared on css with the key _background-image_ or _background_ ( As shown in [here)](https://developer.mozilla.org/en-US/docs/Web/CSS/radial-gradient) ). After this, I tried another approach.

### Workaround
I created a CSS file with the gradient value on it:
```css
.ComponentStyle {
      background: radial-gradient(ellipse farthest-corner at 45px 45px , #00FFFF 0%, rgba(0, 0, 255, 0) 50%, #0000FF 95%);
      height: 100%;
      width: 100%;
      background-position: center;
      background-repeat: no-repeat;
      background-size: cover;
}
```
and add it as a ```className``` to the "Slide" tag like this: 
```jsx
<Slide transition={["fade"]} className = "ComponentStyle">
```

After adding the class I got the gradient background but there's some kind of margin around it. 
<img width="1040" alt="asdsad" src="https://user-images.githubusercontent.com/9245962/43371074-0fa8af16-9350-11e8-9348-9d25782b0259.png">

If you have another workaround, let me know. It would be cool to have a native "property" for the "Slide" tag like ```bgGradient``` but, actually, there's no other way at the moment.

**Comment by ebrillhart** [comment]
closed by #587
